docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -18,7 +18,7 @@ There you can find links to upgrade notes for other versions too.
         -        $filepath = $file->getPathname();
         +        $filepath = TransformString::removeDriveLetterFromPath($file->getPathname());
         ```
-
+- run `php phing standards-fix` so all nullable values will be now defined using nullability (?) symbol ([#1010](https://github.com/shopsys/shopsys/pull/1010))
```

Comment:
This upgrade note belongs to [UPGRADE-v8.0.0.md](https://github.com/shopsys/shopsys/blob/8.0/docs/upgrade/UPGRADE-v8.0.0.md), so we don't have to resolve conflicts on upgrade notes all the time.

Reply:
Changed